cmd/root.go

```diff
@@ -46,11 +60,11 @@ func Execute() {
 }
 
 func init() {
-	cobra.OnInitialize(config.InitConfig)
-	rootCmd.PersistentFlags().StringVar(&config.ConfigFile, "config", "", "config file (default is $HOME/.cpma.yaml)")
+	cobra.OnInitialize()
+	rootCmd.PersistentFlags().StringVar(&env.ConfigFile, "config", "", "config file (default is $HOME/.cpma.yaml)")
```

Comment:
We don't need default config place. I think it's wrong for this tool and what user would expect. Wdyt?

Reply:
@gildub What do think about this?

Reply:
It's not a content for this patch. Feel free to merge.